core/src/main/java/com/klaytn/caver/crypto/KlayCredentials.java

```diff
@@ -87,17 +173,34 @@ public boolean equals(Object o) {
 
         KlayCredentials that = (KlayCredentials) o;
 
-        if (ecKeyPair != null ? !ecKeyPair.equals(that.ecKeyPair) : that.ecKeyPair != null) {
+        if (!Objects.equals(ecKeyPairForTransactionList, that.ecKeyPairForTransactionList)) {
```

Comment:
If there are two `KlayCredentials` objects which have same ecKeyPairs and different order, these objects are not same?

Reply:
Yes. The order of the signatures differs depending on the order of the keys, so different key sequences are determined by different KlayCredentials.